blog/src/__tests__/CreatePost.test.js

```diff
@@ -0,0 +1,37 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { renderWithProviders } from '../utils/wrappertesting';
+import CreatePost from "../components/CreatePost/CreatePost"
+
+describe('Unit tests for CreatePost component', () => {
```

Comment:
Add a test to check if a proper img URL is added. And also don't forget to check the slug input.